### PR TITLE
[RFC] Some suggested cleanup after #1788.

### PR DESCRIPTION
**Pull request by elmart**
@oni-link This implements all your suggestions remaining after #1788.
Note the solution about getdigits(). I'd like some feedback on that.

### CODE REVIEW

**Comment by elmart** [comment]
Travis build fails because getdigits() refactor is breaking a functional test. Probably due to the "abort on failure" approach. Looking at it.

**Comment by elmart** [comment]
Repushed.

**Comment by oni-link** [comment]
@elmart, I commented on a commit that was repushed. Not sure if you could see my comments: 
https://github.com/elmart/neovim/commit/7a0ed9fe41f88df450efb62a9b027faea80cb301#commitcomment-9233201

**Comment by elmart** [comment]
Yes, I got a mail notification.
I've just repushed a last version including the ERANGE check only.

**Comment by elmart** [comment]
Repushed.

**Comment by elmart** [comment]
Repushed.

**Comment by elmart** [comment]
Repushed.

**Comment by oni-link** [comment]
LGTM.

File: src/nvim/charset.c [hunk 1]
```diff
@@ -1676,26 +1676,37 @@ char_u* skiptowhite_esc(char_u *p) {
...
-    ++p;
-  }
-  // skip to next non-digit
-  p = skipdigits(p);
-  *pp = p;
-  return retval;
+/// Get a long number from a string.
+///
+/// A get_digits wrapper restricted to long values.
+long get_long_digits(char_u **pp)
```
Comment (justinmk) [inline src/nvim/charset.c]: For this kind of function "family" I think `getdigits`, `getdigits_int` and `getdigits_long` follows "parallel form" and is more discoverable. What do you think @elmart?

Comment (elmart) [inline src/nvim/charset.c]: I did in fact doubt about that when initially doing it.
In the end, I opted for the more "natural speech" option, but it was probably the wrong option.
You're right the other approach is more discoverable.
I'll rename them.

**Comment by elmart** [comment]
@justinmk Done. I quickly tried to integrate renaming commit into original refactoring commit, but it was not trivial, as I had conflicts because commit in between those just happens to change some things from long to int. So, if you don't mind, I will leave renaming where it is, as an independent commit.

**Comment by justinmk** [comment]
No problem. Thank you both!

**Pull request merged**